src/include/kdump/dialogs.rb

```diff
@@ -531,7 +531,8 @@ def kdump_memory_widget
       else
         _("Kdump Memor&y [MiB]")
       end
-      low_range = format(_("(%{min} - %{max}, default: %{default})"),
+      # TRANSLATORS: Valid range of allocated kdump momory.
```

Comment:
typo: momory.

But I still miss in that comment a note advising the translators to not translate the parts within curly brackets. That's the part the really worries me. I can already foresee a translator doing this (they very often do when we use this ruby-specific syntax for named placeholders):

`"gering: %{gering}; höchste: %{höchste}; Default: %{empfohlen}"`

I know the words are not the right ones, but what I want to illustrate is that many translators tend to translate the variable names in strings line `"something %{var_name}"`. We should be explicit about that when we use that format.